addons/hr/models/res_partner.py

```diff
@@ -32,10 +32,19 @@ def _compute_employees_count(self):
 
     def action_open_employees(self):
         self.ensure_one()
+        if len(self.employee_ids) > 1:
+            return {
+                'name': _('Related Employees'),
+                'type': 'ir.actions.act_window',
+                'res_model': 'hr.employee',
+                'view_mode': 'kanban,tree,form',
+                'domain': [('id', 'in', self.employee_ids.ids)],
+            }
         return {
-            'name': _('Related Employees'),
+            'name': _('Employee'),
             'type': 'ir.actions.act_window',
             'res_model': 'hr.employee',
-            'view_mode': 'kanban,tree,form',
-            'domain': [('id', 'in', self.employee_ids.ids)],
+            'res_id': self.employee_ids.ids,
+            'view_mode': 'form',
+            'context': {'active_id': self.employee_ids.ids},
```

Comment:
No need to pass this context, unless I am missing something :thinking: It should already be a case